inginious/frontend/pages/utils.py

```diff
@@ -187,7 +188,7 @@ def GET(self, *args, **kwargs):
                     and not self.__class__.__name__ == "ProfilePage":
                 return redirect("/preferences/profile")
 
-            if not self.is_lti_page and self.user_manager.session_lti_info() is not None:  # lti session
+            if not self.is_lti_page and self.user_manager.session_lti_info() is not None:  # lti session, TODO(mp): Not sure whether it is still needed
```

Comment:
```suggestion
            if not self.is_lti_page and self.user_manager.session_lti_info() is not None: 
```
I think it is still. There were pages defined not to be accessible through an LTI session. For instance, groups features (even if they should not be by definition), admin pages,...